### PR TITLE
[INJIMOB-780]: Implemented Pop up for downloading VC failure scenario

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
**[INJIMOB-780](https://mosip.atlassian.net/browse/INJIMOB-780)**: Enhancement on Pop-up design and message for downloading VC failure scenario.

Respective screen record

[INJIMOB-780]: https://mosip.atlassian.net/browse/INJIMOB-780?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: screens/Home/MyVcsTab.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {HomeScreenTabProps} from './HomeScreen';
 import {AddVcModal} from './MyVcs/AddVcModal';
 import {GetVcModal} from './MyVcs/GetVcModal';
 import {useTranslation} from 'react-i18next';
-import {GET_INDIVIDUAL_ID} from '../../shared/constants';
+import {BANNER_TYPE_ERROR, GET_INDIVIDUAL_ID} from '../../shared/constants';
 import {
   ErrorMessageOverlay,
```
Comment (Alka1703) [inline screens/Home/MyVcsTab.tsx:12]: Please remove this import

File: screens/Home/MyVcsTab.tsx [hunk 2]
```diff
@@ -374,12 +383,6 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = props => {
         </Row>
       </MessageOverlay>
 
-      <ErrorMessageOverlay
-        translationPath={'MyVcsTab'}
-        isVisible={controller.isSavingFailedInIdle}
-        error={storeErrorTranslationPath}
```
Comment (Alka1703) [inline screens/Home/MyVcsTab.tsx:380]: Please remove 'storeErrorTranslationPath' constant as it is no more getting used

File: screens/Home/MyVcsTab.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import {HomeScreenTabProps} from './HomeScreen';
 import {AddVcModal} from './MyVcs/AddVcModal';
 import {GetVcModal} from './MyVcs/GetVcModal';
 import {useTranslation} from 'react-i18next';
-import {GET_INDIVIDUAL_ID} from '../../shared/constants';
+import {BANNER_TYPE_ERROR, GET_INDIVIDUAL_ID} from '../../shared/constants';
```
Comment (Alka1703) [inline screens/Home/MyVcsTab.tsx:10]: Please merge line 10 and 30 into one

Comment (srikanth716) [reply screens/Home/MyVcsTab.tsx]: merged

File: screens/Home/MyVcsTab.tsx (hunk 2 above)
Comment (srikanth716) [reply screens/Home/MyVcsTab.tsx:380]: removed

File: screens/Home/MyVcsTab.tsx (hunk 1 above)
Comment (Anil-kumar-Majji) [reply screens/Home/MyVcsTab.tsx]: Removed

File: screens/Home/MyVcsTab.tsx (hunk 2 above)
Comment (Anil-kumar-Majji) [reply screens/Home/MyVcsTab.tsx:380]: Removed

**Review by Alka1703 — APPROVED** [review]
lgtm

**Review by swatigoel — APPROVED** [review]

**Pull request merged**